### PR TITLE
CMakeLists.txt cleanup

### PR DESCRIPTION
**Pull request by debris**
requires https://github.com/ethereum/cpp-ethereum/pull/2853

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,10 +33,8 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") AND NOT (CMAKE_CXX_COMPILER_VER
 endif ()
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
-aux_source_directory(./src SRC_LIST)
 file(GLOB_RECURSE SRC_LIST "./plugins/*.cpp" "./src/*.cpp")
```
Comment (chfast) [inline CMakeLists.txt]: Make the naming consistent, e.g. SOURCES and HEADERS.

File: CMakeLists.txt [hunk 2]
```diff
@@ -31,10 +33,8 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") AND NOT (CMAKE_CXX_COMPILER_VER
...
 file(GLOB_RECURSE HEADERS "./plugins/*.h" "./src/*.h")
```
Comment (chfast) [inline CMakeLists.txt]: `./` is not needed.

Comment (chfast) [inline CMakeLists.txt]: Leter the is another `file(GLOB HEADERS ...)`.

**Comment by chfast** [comment]
Can you use target_include_directories instead of include_directories?

File: CMakeLists.txt (hunk 2 above)
Comment (debris) [inline CMakeLists.txt]: > `./` is not needed.

one of the cmake versions (don't remember which one now) has problems if you do not put `./` in path.

> Leter the is another `file(GLOB HEADERS ...)`.

fixed in 5b2e6435825c44b6e94640eda065e1bcab0c52a5

File: CMakeLists.txt (hunk 1 above)
Comment (debris) [inline CMakeLists.txt]: `SRC_LIST` is used in all other targets of cpp-ethereum repo. Also `eth_add_executable` expects `SRC_LIST` to be set. I would prefer to leave the current name for now.

**Comment by debris** [comment]
> Can you use target_include_directories instead of include_directories?

I would prefer to change it in a create separate PR.

File: CMakeLists.txt (hunk 2 above)
Comment (debris) [inline CMakeLists.txt]: > > `./` is not needed.
> 
> one of the cmake versions (don't remember which one now) has problems if you do not put `./` in path.

You are right. It is not required here, cause we use `file(GLOB ..`

**Pull request closed without merging**